app/frontend/schools/settings/components/SchoolCustomize__LinkComponent.res

```diff
@@ -127,18 +127,29 @@ module LinkEditor = {
       {switch kind {
       | HeaderLink
       | FooterLink => <>
-          <input
-            value=state.title
-            required=true
-            autoFocus=true
-            id={"link-title-" ++ id}
-            className=inputClasses
-            placeholder="A short title for a new link"
-            onChange={event => {
-              let value = ReactEvent.Form.target(event)["value"]
-              send(UpdateTitle(value))
-            }}
-          />
+          <div className="flex flex-col gap-1 flex-1">
+            <input
+              value=state.title
+              required=true
+              autoFocus=true
+              id={"link-title-" ++ id}
+              className=inputClasses
+              placeholder="A short title for a new link"
+              onChange={event => {
+                let value = ReactEvent.Form.target(event)["value"]
+                send(UpdateTitle(value))
+              }}
+              maxLength=24
```

Comment:
A curious question, Why did you pick 24?

Reply:
@bodhish 

Because the `create` input for title already had the `max_length` attribute set to `24`. So, I added the same for update and checked with Vinu as well. He said it is fine.